example/ion-sfu-gstreamer-receive/main.go

```diff
@@ -69,7 +69,14 @@ func runClientLoop(addr, session string) {
 
 		codecName := strings.Split(track.Codec().RTPCodecCapability.MimeType, "/")[1]
 		fmt.Printf("Track has started, of type %d: %s \n", track.PayloadType(), codecName)
-		pipeline := gst.CreatePipeline(strings.ToLower(codecName))
+		sink := "fakesink"
+		switch track.Kind() {
+		case webrtc.RTPCodecTypeAudio:
+			sink = "autoaudiosink"
+		case webrtc.RTPCodecTypeVideo:
+			sink = "autovideosink"
+		}
+		pipeline := gst.CreatePipeline(strings.ToLower(codecName), "auto)
```

Comment:
hey, thanks for the fix -- is this last line a typo? looks like there's opening quotes, but no closing quotes, what happened here?

Reply:
Oh weird, not sure what happened, but fixed it.